example/example/urls.py

```diff
@@ -0,0 +1,12 @@
+from django.conf import settings
+from django.conf.urls.static import static
+from django.contrib import admin
+from django.urls import include, path
+
+from . import views
+
+urlpatterns = [
+    path("select2/", include("django_select2.urls")),
+    path("", views.BookCreateView.as_view(), name="book-create"),
+    path('admin/', admin.site.urls),
+] + static(settings.STATIC_URL, document_root=settings.STATIC_ROOT)
```

Comment:
` + static(settings.STATIC_URL, document_root=settings.STATIC_ROOT)` can be dropped since we already have `django.contrib.staticfiles` in `INSTALLED_APPS` which serves `./django_select2/static/` for us. I tried it myself to verify it still works.

Related and could be of interest: https://github.com/django-extensions/django-extensions/issues/1528